src/main/java/com/elixr/poc/Constant/ErrorConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.elixr.poc.Constant;
```

Comment:
Package names should comply with a naming convention

Reply:
updated